NOTES.md

```diff
@@ -3,15 +3,15 @@
 # Release checklist
 
  1. Determine if this should be a major, minor, or patch release.
- 1. Run `bumpversion part`  (where part is major, minor or patch).
```

Comment:
I know this feels weird at first, but I intentionally use `1.` for each of these, because Markdown automatically deals with these.  And more often than not, people will first encounter this file on GitHub, where it shows up correctly.

The bigger reason here is laziness.  Let's suppose I find a new step 2.  Well, now I have to actually go and renumber  2, and all subsequent items.  If I forget, it still would show up correctly in a MD viewer, but then it's inconsistent in a text editor, and that bug me *more* than seeing a silly `1.` everywhere.

https://www.markdownguide.org/basic-syntax/#ordered-lists

BTW, it's worth taking some time to learn MarkDown. It's used many places, although there's no real "standard" so sometimes the exact support varies.  But GitHub uses it for .MD files (like nearly all of our READMEs), GitHub comments, and Slack supports a subset of the basic inline markup.